module/localisation/BallLocalisation/src/BallModel.hpp

```diff
@@ -73,7 +73,9 @@ namespace module::localisation {
 
             const Eigen::Matrix<Scalar, 3, 1> rBWw(state[PX], state[PY], field.ball_radius);
             const Eigen::Matrix<Scalar, 3, 1> rBCc_cart(Eigen::Affine3d(Hcw) * rBWw);
```

Comment:
These variable names could be more descriptive.

Reply:
https://nubook.nubots.net/system/foundations/mathematics#vectors

Reply:
These names are literally the most descriptive!
Sometimes it's useful to define what each letter does if it's not one f our common ones